depthai_sdk/src/depthai_sdk/classes/output_config.py

```diff
@@ -176,7 +177,7 @@ def setup(self, pipeline: dai.Pipeline, device: dai.Device, _) -> List[XoutBase]
             xoutbase.setup_base(self.new_packet)
             xouts.append(xoutbase)
 
-            if self.visualizer:
-                xoutbase.setup_visualize(self.visualizer, xoutbase.name)
+            if hasattr(xoutbase, 'setup_visualize'):
```

Comment:
@daniilpastukhov - I'm not exactly sure, but why do we have visualizers inside the `oak.sync()`?